repos/team.toml

```diff
@@ -0,0 +1,8 @@
+org = "rust-lang"
+name = "team"
+description = "Rust teams structure"
+bots = ["bors", "highfive", "rustbot", "rust-timer"]
+
+[access.teams]
+core = "admin"
+mods = "maintain"
```

Comment:
I think the access + bots definitions here are OK for now, but I do think we may want some kind of "standard template" for repositories that you can opt out of. For example, rustbot + highfive probably should have access to anything in the org.